h2o-core/src/main/java/hex/ModelBuilder.java

```diff
@@ -62,6 +62,13 @@ protected boolean stop_requested() {
     return _job.stop_requested() || timeout();
   }
 
+  protected long remainingTimeSecs() {
+    long remainingTimeMillis = _build_model_countdown.remainingTime();
+    if (remainingTimeMillis == Long.MAX_VALUE)
+      return 0;
```

Comment:
not sure this is worth converting it back to 0, we're talking about a real lot of millis here... a few million years.
And when consuming this, you'll have to handle this 0 as a special case again.

Reply:
Fixed, thanks! I misread the behaviour of `_build_model_countdown.remainingTime()` in a way where I assume that if there is 0 or less remaining time, it would return `Long.MAX_VALUE`...